### PR TITLE
docs: update readme with new links and restructure a bit

### PR DESCRIPTION
**Pull request by braddialpad**
Update the readme with new request / report links and restructure to be a bit more similar to the dialtone readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,3 +64,20 @@ If you would like to contribute to Dialtone Vue the first step is to get the pro
 4. Visit local dev server at http://localhost:9011/
 
 Next read the more detailed contributor documentation in [CONTRIBUTING.md](.github/CONTRIBUTING.md).
+
+## Project Status
+
+Dialtone Vue is a new project, and as such it is under constant development as we add new components and refine existing ones. Please refer to the [jira board][jira] to see what we are currently working on and what we have planned for the future.
+
+[jira]: https://switchcomm.atlassian.net/secure/RapidBoard.jspa?rapidView=246
```
Comment (josedialpad) [inline README.md]: This board "Component Library" link is outdated?

```suggestion
[jira]: https://dialpad.atlassian.net/browse/DT
```

**Review by josedialpad — APPROVED** [review]
w/ one suggestion

File: README.md [hunk 2]
```diff
@@ -72,3 +64,20 @@ If you would like to contribute to Dialtone Vue the first step is to get the pro
...
+[request]: https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10901&pid=12428
+[report]: https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10878&pid=12428
+
+## Requesting features / reporting bugs
+
+Requesting a feature or reporting a bug? Please do so at the below links:
+
+[Request Feature](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10901&pid=12428)
+[Report Bug](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10878&pid=12428)
```
Comment (josedialpad) [inline README.md]: nit:

```suggestion
- [Request Feature](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10901&pid=12428)
- [Report Bug](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10878&pid=12428)
```

File: README.md (hunk 1 above)
Comment (braddialpad) [reply README.md]: great find thanks!

**Pull request merged**